TOC-tidb-cloud.md

```diff
@@ -400,6 +401,7 @@
         - [`USER_PRIVILEGES`](/information-schema/information-schema-user-privileges.md)
         - [`VIEWS`](/information-schema/information-schema-views.md)
   - [System Variables](/system-variables.md)
+  - [API Reference](https://docs.pingcap.com/tidbcloud/api/v1beta)
```

Comment:
We can also put this on our doc home page: https://docs.pingcap.com/tidbcloud/

Reply:
Done in https://github.com/pingcap/docs/pull/10049